### PR TITLE
[CHORE] Add Deployment Script

### PR DESCRIPTION
**Pull request by mint-chocolate-frappuccino**
## Description
This is a PR containing
- adding/altering deployment scripts for
   - adding LatteVault deployment on 001
   - adding addLiquidity (003) deployment script
   - adding an ownership transfer to TimeLock deployment script
   - adding a deployment script for adding a master barista pool
- add more unit tests on LatteSwapRouter
- remove admin modifier since it's redundant with an `owner` role
## Purpose of this change
- to fulfill an AMM flow and ensure that no defects detected
## Is this PR Breaking Changes?
- [ ] Y
- [x] N
## Changes:
- chore: add solidity settings
- refactor: remove onlyAdmin modifier since it's kinda redundant
- chore: add LatteVault deployment on master barista deployment script
- chore: add addLiquidity deployment script for adding an initial liquidity
- chore: add transfer ownership to TimeLock deployment script
- chore: add a deployment script for adding a pool to master barista
- test: rename a test description for latte swap factory test
- test: makes chainID as an injected param rather than 1 which is hard-code
- test: add a unit test for latte swap router
- chore: update package json
- chore: add script command for s000, s001

### CODE REVIEW

File: deploy/001_deploy_master_barista.ts [hunk 1]
```diff
@@ -14,14 +14,22 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
...
-  const LATTE_END_RELEASE = '';
   const LATTE_PER_BLOCK = ''
   const LATTE_START_BLOCK = ''
+
+  // LATTE token
   const GOVERNOR_ADDRESS = ''
+  const LATTE_START_RELEASE = '';
+  const LATTE_END_RELEASE = '';
+
+  // LatteVault
```
Comment (coco-coke-kub) [inline deploy/001_deploy_master_barista.ts:30]: Separate `LatteVault` to another deployment script. We won't launch it at Genesis.

File: deploy/S001_timelock_add_pool_to_master_barista.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+
+
+
+
+
+  const config = network.name === "mainnet" ? MainnetConfig : TestnetConfig
+  const allAvailableStakingToken = [...config.LpTokens]
+  if (!verifyStakingTokenAddress(allAvailableStakingToken, STAKING_TOKEN_ADDRESS)) throw new Error('verifyStakingTokenAddress:: not found')
```
Comment (coco-coke-kub) [inline deploy/S001_timelock_add_pool_to_master_barista.ts:42]: verify like this won't work as there can be 1,000++ LPs and I don't think we will add all of them in `config.LpTokens`. I think can leave it out without any validation.

**Review by coco-coke-kub — CHANGES_REQUESTED** [review]

File: deploy/001_deploy_master_barista.ts (hunk 1 above)
Comment (mint-chocolate-frappuccino) [reply deploy/001_deploy_master_barista.ts:30]: ok

File: deploy/S001_timelock_add_pool_to_master_barista.ts (hunk 2 above)
Comment (mint-chocolate-frappuccino) [reply deploy/S001_timelock_add_pool_to_master_barista.ts:42]: that's unfortunated, ok

**Review by coco-coke-kub — APPROVED** [review]

**Pull request merged**